cmake/DGtalConfig.cmake.in

```diff
@@ -157,6 +157,25 @@ IF(@CGAL_FOUND_DGTAL@)
   SET(WITH_CGAL 1)
 ENDIF(@CGAL_FOUND_DGTAL@)
 
+IF(@FFTW3_FOUND_DGTAL@)
+  FIND_PACKAGE(FFTW3 REQUIRED)
```

Comment:
When you have "found" the libraries/includes, you shouldn't retry a FIND_PACKAGE in the DGtalConfig (see MAGICK section or Cairo). 
Path are known and are not supposed to change between the DGtal build and the external project you're compiling.

In my system, there is no FindFFTW3.cmake in the cmake module repository so you may need to ship if with DGtal (which is not what you want).

For some deps, we force a FIND_PACKAGE because the package set specific variables..